### ISSUE TITLE
Connection Exception

### ISSUE THREAD

**Issue by ordinaryorange** [OYIon/LiveSharp#69]
Just downloaded livesharp today for a trial with a Blazor wasm project.

I'm getting errors in the output window of VS. Here is a snippet.

>livesharp: Transport type: LiveSharp.Support.Blazor.BlazorTransport
livesharp: Connecting to host 192.168.1.220:50540
livesharp: Activating UpdateHandler BlazorUpdateHandler
Blazor update handler started
livesharp: error: Connection exception: System.AggregateException: One or more errors occurred. (TypeError: Failed to fetch) ---> WebAssembly.JSException: TypeError: Failed to fetch
  at System.Net.Http.WebAssemblyHttpHandler.doFetch (System.Threading.Tasks.TaskCompletionSource`1[TResult] tcs, System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) <0x319ed70 + 0x00a30> in <filename unknown>:0 
  at System.Net.Http.WebAssemblyHttpHandler.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) <0x31796e0 + 0x00174> in <filename unknown>:0 
  at Microsoft.AspNetCore.Http.Connections.Client.Internal.AccessTokenHttpMessageHandler.SendAsync (System.Net.Http.HttpRequestMessage request, System.Threading.CancellationToken cancellationToken) <0x3178698 + 0x001b6> in <filename unknown>:0 

Livesharp then seems to go around the grounds to connect to the server on the other 3 ports. But the sever is only listening on 50540
Ideas ?

**Comment by ionoy** [comment]
Hello @ordinaryorange 

Thank you for the feedback. Can you check the connection error in Chrome developer tools?

You should be able to find it here

![image](https://user-images.githubusercontent.com/592624/82867618-daba1c80-9f33-11ea-977e-e430c63eef7f.png)

**Comment by ordinaryorange** [comment]
The request is 

> https://localhost.livesharp.net:56539/livesharp/negotiate?negotiateVersion=1

which is failing as that port is not open on the server. The server only has TCP 50540, 50539, 27621 listening.

**Comment by ionoy** [comment]
Client application cycles through these ports `50539, 52539, 54539, 56539, 58539` so you should be able to see the connection to `56539` at some point.

**Comment by ordinaryorange** [comment]
I can see the port cycling in the errors in the output window. These cycle over the 5#540 ports. However the browser connections never change

**Comment by ionoy** [comment]
`5#540` is a bit misleading, since it is changed to `5#539` in Blazor transport. The HTTPS connection is Socket port - 1.

**Comment by ionoy** [comment]
You will probably see the request to `https://localhost.livesharp.net:50539/livesharp/negotiate?negotiateVersion=1` if you refresh the browser window.

**Comment by ordinaryorange** [comment]
I've rebooted things, and now the client is cycling through the ports. But still no connection from the client to 50539.

I just installed the server to a fresh VM to see what I would get. Once running the server listens on 50540 and 50539, the same as my development machine. So that seems normal. 
The 40 port seems to be for the "server" end and 39 for client connections ?

I can redirect the client requests to the VM by adding localhost.livesharp.net to my hosts, but is there a way to redirect the server end to an alternate IP ? The output window is still showing connection attempts my my LAN IP.

Just an idea for a test.
Or unless you want me to test something else ?
Can gitter too if easier.
